tox.ini

```diff
@@ -267,8 +273,8 @@ max-line-length = 88
 [travis]
 python =
   2.6: py26,custom
-  2.7: py27,flake8,pylint,custom
-  3.5: coveralls,custom
-  3.6: py36,black,yamllint,custom
+  2.7: py27,coveralls,flake8,pylint,custom
```

Comment:
Is this change correct? I believe this change was part of another sync attempt and it does not make sense in the current configuration since coveralls needs to run only once.

Reply:
I'll change `coveralls` to be executed only in the 3.6 environment.

@tyll Also - there is a new `.travis/runcoveralls.sh` in the template - can this be used to replace the network
```
    bash -c 'cd ..; bash .travis/fix-coverage.sh; cd -'
    coveralls
```
?
It looks as though there is some creation and fixup of .coveragerc in template `.travis/runcoveralls.sh` but I don't know if this is the same as what network is doing, or if we need some additional work to template `.travis/runcoveralls.sh` in order to be suitable for network.